lib/network_utils.pm

```diff
@@ -40,7 +40,8 @@ sub setup_static_network {
     $args{silent} //= 0;
     configure_static_dns(get_host_resolv_conf(), silent => $args{silent});
     assert_script_run('echo default ' . $args{gw} . ' - - > /etc/sysconfig/network/routes');
-    my $iface = iface();
+    $args{iface} //= iface();
```

Comment:
usually `iface()` function perfectly detecting available interfaces

Reply:
I remember that I had to specify this, but was long time ago, I can test it without this change, to see if the problem appears again

Reply:
done